operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -6,16 +6,14 @@ package v1beta1
 import (
 	"encoding/json"
 	"fmt"
-	"os"
-
 	"github.com/Jeffail/gabs"
+	"github.com/datastax/cass-operator/operator/pkg/serverconfig"
+	"github.com/datastax/cass-operator/operator/pkg/utils"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
-
-	"github.com/datastax/cass-operator/operator/pkg/serverconfig"
-	"github.com/datastax/cass-operator/operator/pkg/utils"
+	"os"
```

Comment:
nit: this import switch should be reverted

Reply:
@jimdickinson by any chance are there any project style settings that I can import into GoLand? Or are there style guidelines available so that I can configure GoLand to be consistent?